app/assets/stylesheets/views/_covid.scss

```diff
@@ -51,27 +51,19 @@
   }
 }
 
-.covid__page-header-announcement {
-  margin-bottom: govuk-spacing(7);
-
-  @include govuk-media-query($from: desktop) {
-    margin-bottom: govuk-spacing(3);
-  }
-
-  .govuk-body {
-    margin-bottom: govuk-spacing(2);
-  }
+.covid__announcements {
+  margin-top: govuk-spacing(5);
 
   .covid__announcement-published-text {
-    @include govuk-font(16);
+    margin-top: govuk-spacing(2);
   }
 }
 
 .covid__business-list-wrapper {
   margin-bottom: 45px;
 }
 
-.covid__logo-wrapper {
+.covid__action-link-wrapper {
   margin-top: govuk-spacing(5);
   margin-bottom: govuk-spacing(8);
```

Comment:
An alternative to adding this margin bottom would be to modify the padding bottom of `.covid__page-header-light` - that way the spacing is controlled by one thing only and is simpler to read.